sklearn/gaussian_process/correlation_models.py

```diff
@@ -12,7 +12,7 @@
 import numpy as np
 
 
-def absolute_exponential(theta, d):
+def absolute_exponential(theta, dx):
```

Comment:
This seem to have been a typo from [version 0.10 docs](http://scikit-learn.org/0.10/modules/generated/sklearn.gaussian_process.correlation_models.absolute_exponential.html)